pages/services/include/beta-advanced.tmpl

```diff
@@ -0,0 +1,396 @@
+# Advanced
```

Comment:
OK, I see why you felt you needed to create a whole new #include file for this set of documentation. However, you added only **two** lines to the 395 lines in the #advanced.tmpl file. This really does not justify the creation of an entirely new #include file, which is 99% the same as the former one; this will create confusion among other authors. That's why we restrict creation of new #include files to the docs team.

Since you still need 99% of the original #advanced.tmp file, I suggest this: 

1. Copy and paste the entire contents of the #advanced.tmpl file into your pages/services/beta-cassandra/2.5.0-3.0.16-beta/advanced/index.md file.
2. Then add the required changes to the system requirements that you added to this beta-advanced.tmpl file. 
3. Delete this beta-advanced.tmpl file and re-commit the pages/services/beta-cassandra/2.5.0-3.0.16-beta/advanced/index.md file.
I should tell you that the docs team is planning to complete revise the #include files for these documents in the future, which will make them easier to modify/customize as you needed to do. Until that time, however, please go ahead and copy/paste the contents of the files and alter them, rather than editing or creating new #include files.